client/layout/logged-out.jsx

```diff
@@ -4,7 +4,7 @@ import classNames from 'classnames';
 import { localize } from 'i18n-calypso';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
-import GdprBanner from 'calypso/blocks/gdpr-banner';
```

Comment:
Are we planning to cleanup the unused code that's part of the original gdpr banner?

Reply:
The old GdprBanner code has already been removed as part of this PR ([eed33f8](https://github.com/Automattic/wp-calypso/pull/69304/commits/eed33f88c00a60abf2ec75a9f6899ec32f38fbdb))